entailments/imce.jpl.nasa.gov/foundation/behavior/behavior/catalog-v001.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file should be removed from the Git. It's created by opening a file in the same directory with Protege. Not something we want to put under CM.

Reply:
Sure - I can remove it. Are you sure it's only Protege? We did not use Protege at all - we did, however, run the ontology processing workflow locally. I noticed other, similar files in git, so I initially thought it was OK to include.

Reply:
Done in latest commit